### ISSUE TITLE
Shell command - cat not working for huge files during vault write operation

### ISSUE THREAD

**Issue by jagpreetsinghsasan** [hyperledger-labs/blockchain-automation-framework#1642]
**Describe the bug**
According to the [ongoing discussion](https://chat.hyperledger.org/channel/blockchain-automation-framework?msg=cFDh78k3H3TnXtTB7) on the rocket chat, it seems the cat command is not working for huge files for vault write operations. Mostly this is due to huge genesis file being created (185k)

**To Reproduce**
1. Generate a file with size more than 185k
2. Run this sort of [command](https://github.com/jagpreetsinghsasan/blockchain-automation-framework/blob/cf3b9ef90b5408af8b5d40f319d0c89e177824df/platforms/hyperledger-fabric/configuration/roles/create/genesis/tasks/main.yaml#L23) with cat of the above created file 

**Expected behavior**
The file should be written onto the vault

**Additional context**
This is just an assumption, as per reference to similar issues found over the net.
If this ain't the issue, this needs a root cause analysis to find the actual issue.

**Comment by kaverikhaneja** [comment]
I can work on this.

**Comment by SoundaryaA3098** [comment]
Hi All,

I have tested by modifying the "vault write" command, it worked and the genesis.block file got written into the vault successfully. I have modified the code in "Write genesis block to Vault" under roles/create/genesis like

![Screenshot from 2021-09-01 19-04-28](https://user-images.githubusercontent.com/59388030/131682003-22ed2d7a-3b6a-475a-9048-bdada8f4ee46.png)

All the mentioned number of channels got created. But the joinchannel jobs are not coming up. The result of "kubectl describe helmrelease joinchannel-peer0-join-limaonechannel-org1-peer0 -n org1-net" gives the following event,

**"synchronization of release 'joinchannel-peer0-join-limaonechannel-org1-peer0' in namespace 'org1-net' failed: failed to determine sync action for release: failed to retrieve Helm release: releaseContent: Release name is invalid: joinchannel-peer0-join-limaonechannel-org1-peer0"**

And I am facing these bugs of joinchannel job and anchor peer updation jobs failing in BAF v0.9.0.0 release sometimes. I have tested multiple channels with BAF v0.9.0.0 release by configuring 2 channels. In that case, all the channel related jobs like createchannel, joinchannel came up successfully and anchor peer updation was also done. But when I tested with more number of channels (eg: 6 or 9), there is an issue with joinchannel and anchor peer updation jobs.
